go.sum

```diff
@@ -0,0 +1,2 @@
+golang.org/x/sys v0.0.0-20201119102817-f84b799fce68 h1:nxC68pudNYkKU6jWhgrqdreuFiOQWj1Fs7T3VrH4Pjw=
```

Comment:
Same comment as the go.mod. It would be best to avoid the go.sum altogether unless absolutely needed.

Reply:
same as go.mod